source/less/blocks/recall-form.less

```
@@ -0,0 +1,116 @@
+.recall-form__info {
+  padding: 0;
+  margin: 0;
+  border: none;
+  padding: 24px 20px 34px;
+}
+
+.recall-form__title {
+  font-family: "PT Sana", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 26px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #000000;
+  padding: 0;
+}
+
+.recall-form-question {
```

```suggestion
.recall-form__question {
```
Это класс элемента, а не блока.